### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by MrSayyod**
- Used `create-react-app` to initialize a new React app.
- Added `prop-types` using `npm` for typechecking props.
- Added `big.js` using `npm`.
- Created a Heroku app.
- Deployed to Heroku.
- Created a dedicated repository for the project.
- Set up the code linters.
- Set the development branch as a default branch in the repository.
- Committed changes often.

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work setting up the project. Well done.
### changes 🔕 🔕 🔕 
- update the readme. Add a description of the app, sections for getting started, installing packages, running locally, author, contributing, etc. These sections make your repo more professional
- delete unnecessary files such as the logo.svg
- deploy the app and add a live link to the readme file

**Comment by MrSayyod** [comment]
- Updated the readme.
- Added a description of the app, sections for getting started, installing packages, running locally, author, contributing, etc.
- Added a live link to the readme file
- Deleted unnecessary logo.svg
- Deployed the app

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```
Comment (george-shammar) [inline src/reportWebVitals.js:8]: - [ ] Thanks for making some the required changes. You still have multiple unneccessary files that are not related to the calculator app. e,g `logo192.png`, `logo152.png`, `favicon`, `reportWebVitals.js` e.t.c. Please remove any file that would not be directly of any use in the project.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @MrSayyod , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by MrSayyod** [comment]
- Deleted unnecessary files

**Review by whiz25 — CHANGES_REQUESTED** [review]
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes. All unnecessary files have been removed. Well done!
### changes 🔕 🔕 🔕 
- fix linter errors

### Martin Afani
#### Slack: whiz25

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

 - [x] Your app should run correctly with `npm start`:



_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by MrSayyod** [comment]
- Deleted unnecessary files. Now `npm start` works correctly.

**Review by rukundoeric — APPROVED** [review]
Hi @MrSayyod,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**